site/content/docs/11-deploying.md

```diff
@@ -50,3 +50,71 @@ vercel -e SAPPER_TIMESTAMP=$(date +%s%3N)
 ```
 
 [Vercel]: https://vercel.com/home
+
+
+## Deploying with Docker
+
+[Docker](https://docs.docker.com/install/) can be used to build an image and run it on your local system. Doing this locally allows testing the build process before building and deploying it on a cloud platform. A Docker image can be built by adding a Dockerfile to your project:
+
+```Dockerfile
+# Dockerfile
+FROM mhart/alpine-node:12 as build
+
+WORKDIR /app
+COPY . .
+RUN npm ci
+RUN npm run build
+
+FROM mhart/alpine-node:12 as prod
+
+WORKDIR /app
+COPY package.json package-lock.json ./
+RUN npm ci --production
+
+FROM mhart/alpine-node:slim-12
+
+WORKDIR /app
+COPY --from=build /app/__sapper__/build __sapper__/build
+COPY --from=build /app/static static
+COPY --from=prod /app .
+
+CMD ["node", "__sapper__/build"]
+```
+
+This Dockerfile is a [multi-stage build](https://docs.docker.com/develop/develop-images/multistage-build/) that performs three steps:
+
+- Perform a `sapper` build that uses the devDependencies (like the `svelte` compiler itself) to generate minimal code
+- Perform an `npm install --production` that only installs runtime dependencies (like `sirv` and `polka`)
+- Aggregate the build artifacts and the runtime dependencies into the final image
+
+The three steps ensure that the final image will only contain the generated code and the runtime dependencies for production. The final image is based upon [mhart/alpine-node](https://hub.docker.com/r/mhart/alpine-node/) which is a small linux image that uses Alpine and Node.
+
+
+### Docker commands
+
+Commands available to build and run a local Docker image listed below.
+
+```bash
+export IMAGE=sapper-image
+export CONTAINER=sapper
+
+npm install
+# build a docker image
+docker build -t $IMAGE .
+# show image size (sapper-template is 47.1MB)
+docker images $IMAGE --format {{.Size}}
```

Comment:
Most of the commands here don't relate to building and deploying a docker image. This looks more like a docker tutorial.

It would be good to trim these down into the minimum commands to build an image and push it to Dockerhub.